src/app/host/[room]/page.tsx

```diff
@@ -0,0 +1,59 @@
+// HOST ROOM
+'use client'
+
+import { useEffect, useState } from 'react'
+import { socket } from '@/socket'
+import SocketIoTestSection from '@/components/socketIoTest/SocketIoTestSection'
+import HostSettingsButton from '@/components/HostSettingsButton'
+import HostControls from '@/components/HostControls'
+import StoryPoints from '@/components/StoryPoints'
+import CardsContainer from '@/components/CardsContainer'
+
+export type ChatEvent = {
+	message: string
+	timeStamp: string
+	room: string
+}
+
+export default function HostRooms({ params }: { params: { room: string } }) {
+	const [roomId, setRoomId] = useState('')
+	const [userName, setUserName] = useState('')
+
+	// biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
```

Comment:
todo: No, no, never ever ignore `useExhaustiveDependencies` Why did you use it?

Reply:
Because this should only run once, and it was being updated and switching the socket connection. However, I have refactored this to include the dependencies and gotten the socket connection to not update. New PR will fix that.

Reply:
New commits have resolved this.